Dockerfile.assisted-service

```diff
@@ -37,6 +37,7 @@ RUN cd cmd && CGO_ENABLED=1 GOFLAGS="" GO111MODULE=on go build -o /build/assiste
 RUN cd ./cmd/operator && CGO_ENABLED=1 GOFLAGS="" GO111MODULE=on go build -o /build/assisted-service-operator
 RUN cd ./cmd/webadmission && CGO_ENABLED=1 GOFLAGS="" GO111MODULE=on go build -o /build/assisted-service-admission
 RUN cd ./cmd/agentbasedinstaller/client && CGO_ENABLED=1 GOFLAGS="" GO111MODULE=on go build -o /build/agent-installer-client
+RUN cd ./cmd/wiremock && CGO_ENABLED=1 GOFLAGS="" GO111MODULE=on go build -o /build/add-wiremock-stubs
```

Comment:
It's a bit non-conservative to include a binary for tests purposes in the production image. Not sure if we can easily avoid it though, but maybe just worth thinking about alternatives.

Reply:
I don't think it would be complicated, but I don't think it worth the maintaining of another image

Reply:
We're already maintaining the image we're deploying for wiremock (or should be at least).
[This](https://github.com/openshift/assisted-service/blob/3d14207ff4155d3f982d06fd485c828b0f0594f4/deploy/wiremock/wiremock-deployment.yaml#L18) shows it's using `ocpmetal` repo which IIRC is something we used to maintain.

Reply:
FWIW I tried to get rid of all of these in b8aa0b95615847c42434c46b6a08678b3d5d4476 but I guess I missed some

Reply:
Also I'm not sure why converting to python is the only option.

In the past I was able to initialize the wiremock instance using a small go program. Could we not do the same as a part of deploying wiremock (before we start assisted-service)?

Example:
```
package main

import (
	"log"

	"github.com/kelseyhightower/envconfig"
	"github.com/openshift/assisted-service/subsystem"
)

var Options struct {
	OCMHost   string `envconfig:"OCM_HOST"`
	TestToken string `envconfig:"TEST_TOKEN"`
}

func main() {
	if err := envconfig.Process("subsystem", &Options); err != nil {
		log.Fatal(err.Error())
	}

	wiremock := &subsystem.WireMock{
		OCMHost:   Options.OCMHost,
		TestToken: Options.TestToken,
	}

	if err := wiremock.DeleteAllWiremockStubs(); err != nil {
		log.Fatalf("Failed to delete all wiremock stubs, %v", err)
	}

	if err := wiremock.CreateWiremockStubsForOCM(); err != nil {
		log.Fatalf("Failed to init wiremock stubs, %v", err)
	}
}
```

Reply:
@carbonin IIUC this is what I did exactly

Reply:
Why not do it from outside the assisted service image?

Reply:
Lets try to just run the go code to initialize the mocks from the Makefile.
If that doesn't work we can explore other options.